diagnostic.md

```diff
@@ -6,28 +6,28 @@ Write your answers inside this file, where it's indicated by the comments.
 What command do we use to do that? Use the correct URL for your fork of this
 repository in your answer.
 
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:kpie89/git-basics-diagnostic.git
 
 2.  What do you do after cloning a repository, but before starting work?
 
-<!-- Remove this comment and place your answer here. -->
+git init and create a new branch
 
 3.  What command do we use to create a new branch? Name this branch `response`
 in your answer.
 
-<!-- Remove this comment and place your answer here. -->
+git checkout -b <response>
```

Comment:
No `<` or `>`  - that was just there to indicate that it was a placeholder. Otherwise correct.